Snowflake.Data.Tests/SFConnectionPoolT.cs

```diff
@@ -504,6 +504,98 @@ public void TestConcurrentConnectionPoolingAsync()
             ConcurrentPoolingAsyncHelper(connStr, true);
         }
 
+        [Test]
+        public void TestRollbackTransactionOnPooledWhenExceptionOccurred()
+        {
+            SnowflakeDbConnectionPool.SetPooling(true);
```

Comment:
those parameters are static? who cleans them between test methods?

Reply:
Pooling is turned on by default. Maybe we should use SetUp and TearDown to store are restore pooling attribute?